.evergreen/config.yml

```diff
@@ -1042,17 +1042,28 @@ buildvariants:
     tasks:
       - name: "test-mlaunch"
 
-  - matrix_name: "stress"
+  - matrix_name: "stress older"
```

Comment:
Any suggestions on what to name this? I for sure want to take out the space and replace it with an _ but before I restart the CI does anyone have any better ideas?

Reply:
Oleg mentioned that this name is fine.